addons/account_qr_code_emv/tests/test_emv_qr.py

```diff
@@ -0,0 +1,37 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Better to remove this test as it will only fail with the right things installed...